### PR TITLE
config/imxrt1020-evk/ble: add ble config for imxrt1020

### PR DESCRIPTION
**Pull request by dh79pyun**

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [b104b0fea1d790952a55f30b0ca7b39d34b3d517] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b104b0fea1d790952a55f30b0ca7b39d34b3d517] - Code Rule Check OK.

File: build/configs/imxrt1020-evk/ble/Make.defs [hunk 1]
```diff
@@ -0,0 +1,131 @@
+###########################################################################
+#
+# Copyright 2019 NXP Semiconductors All Rights Reserved.
```
Comment (sunghan-chang) [inline build/configs/imxrt1020-evk/ble/Make.defs]: NXP -> Samsung

**Comment by seinfra** [comment]
Target : [2738a856a417d702145b28c23431f9128413b584] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [2738a856a417d702145b28c23431f9128413b584] - Code Rule Check OK.

**Pull request merged**